src/ArcgisAccount/accountManagerStorageUtils.js

```diff
@@ -0,0 +1,160 @@
+import { UserSession } from '@esri/arcgis-rest-auth';
```

Comment:
`@esri/arcgis-rest-auth` needs to be added as a dependency in package.json (this is what's causing the [Netlify build to fail](https://app.netlify.com/sites/calcite-react/deploys/5f2d629c1bf42a0007b6c9df))